spec/takeaway_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'takeaway'
+
+describe Takeaway do
+  it 'should respond to show menu' do 
+    expect(subject).to respond_to(:menu)
+  end
+
+  it 'should call display method from Menu' do
+    expect(subject.menu).to receive(:show)
+    subject.show_menu
+  end
+
```

Comment:
again, already thinking about testing injection (looking at commit 1) here is very switched on. nice. took a lot of research/cheating for me to come to tests like this